common/src/main/java/com/microsoft/identity/common/logging/Logger.java

```diff
@@ -216,18 +216,6 @@ public void log(String tag,
                 });
     }
 
-    /**
-     * Get only the required metadata from the DiagnosticContext
-     * to plug it in the log lines.
-     * Here we are considering the correlation_id and the thread_name.
-     * The need for this is because DiagnosticContext contains additional metadata which is not always required to be logged.
-     *
-     * @return String The concatenation of thread_name and correlation_id to serve as the required metadata in the log lines.
-     */
-    public static String getDiagnosticContextMetadata() {
```

Comment:
I am not sure if any of our partner/consumer is using these or not, as these being public static. (Likely not).
To be on safer side, let's keep them and mark as deprecated.